auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -507,12 +507,6 @@ export default class EntityManagement extends Mixins(AccountChangeMixin, NextPag
   close () {
     this.$refs.errorDialog.close()
   }
-
-  private goToManageBusinesses (isNumberedCompanyRequest: boolean = true): void {
```

Comment:
This method does the exact same thing as startNumberedCompany except it wouldn't refresh the data table.